myia/abstract/infer.py

```diff
@@ -27,6 +27,23 @@
     bind, type_to_abstract
 
 
+class ArrayWrapper:
```

Comment:
Add an import in `myia/abstract/__init__.py` so that we can do `from myia.abstract import ArrayWrapper`.